spec/builders/messages/instagram/message_builder_spec.rb

```diff
@@ -5,12 +5,14 @@
 
   before do
     stub_request(:post, /graph.facebook.com/)
+    stub_request(:get, 'https://imagekit.io/blog/content/images/2020/05/media_library.jpeg')
```

Comment:
can we replace this url with `https://example.com/.....` ?

Reply:
We have used it for story mention attachments to test actual attachments through story mention in [instagram_event_job_spec.rb](https://github.com/chatwoot/chatwoot/blob/4375a7646ee84a20cfd055e722b3aca68e5b34b0/spec/jobs/webhooks/instagram_events_job_spec.rb)

Reply:
It would be better to leave out third-party references in the code which doesn't have an affiliation with us. 

let's replace the URLs used in 

https://github.com/chatwoot/chatwoot/blob/4375a7646ee84a20cfd055e722b3aca68e5b34b0/spec/factories/instagram/instagram_message_create_event.rb#L107

https://github.com/chatwoot/chatwoot/blob/4375a7646ee84a20cfd055e722b3aca68e5b34b0/spec/factories/instagram/instagram_message_create_event.rb#L141

since  webmock would prevent our specs from accessing outside network, we can stick  with something like `example.com` or `local.host` etc.